lib/bridge/strain.rb

```diff
@@ -36,6 +36,21 @@ def self.minors
       [Club, Diamond]
     end
 
+    @mapping = {}
+    %w(Clubs Diamonds Hearts Spades NoTrumps
```

Comment:
Ha!  I was trying to demonstrate I can, actually, contribute while conforming to the surrounding code's existing style.

One book-learnin' thing I've been gnawing on was this one from Robert C. Martin: if I find myself fiddling with horizontal alignment, the problem is the length of the lists, not the lack of alignment.